src/views/Home/Home.vue

```diff
@@ -24,19 +24,19 @@
       </div>
     </div>
     <div class="fotosRandom">
-      <div class="carousel--container">
-        <Carousel :slides="carouselImages" />
+      <div class="carousel--container" >Carousel
+        <carouselHome :slides="carouselImages" />
       </div>
       <!-- TO-DO: DISEÑO DE BOTONES -->
-      <router-link :to="viewMoreUrl" class="ver-mas">Ver catálogo</router-link>  
+      <router-link :to="viewMoreUrl" class="ver-mas" style="font-family: 'Helvetica-SOS'">Ver catálogo</router-link>  
     </div>
     <Footer />
   </div>
 </template>
 
 <script>
 import Footer from "@/components/Footer/Footer.vue";
-import Carousel from "@/components/Carousel/Carousel.vue";
+import carouselHome from "@/components/Carousel/carouselHome.vue";
```

Comment:
Deberia ser CarouselHome
```suggestion
import carouselHome from "@/components/Carousel/CarouselHome.vue";
```